extension.toml

```diff
@@ -1,9 +1,14 @@
 id = "dart"
 name = "Dart"
 description = "Dart support."
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
Please remove the version bump from this PR.

We bump the versions separately.

Reply:
Done